pkgs/applications/networking/browsers/nyxt/default.nix

```diff
@@ -12,36 +22,62 @@ stdenv.mkDerivation rec {
 
   src = lispPackages.nyxt;
 
-  nativeBuildInputs = [ makeWrapper wrapGAppsHook ];
-  gstBuildInputs = with gst_all_1; [
-    gstreamer gst-libav
+  nativeBuildInputs = [ wrapGAppsHook ];
+  buildInputs = [
+    cairo
+    enchant
+    gdk-pixbuf
+    glib
+    glib-networking
+    gobject-introspection
+    gsettings-desktop-schemas
+    gtk3
+    mime-types
+    pango
+  ] ++ (with gst_all_1; [
+    gstreamer
+    gst-libav
     gst-plugins-base
     gst-plugins-good
     gst-plugins-bad
     gst-plugins-ugly
-  ];
-  buildInputs = [
-    glib gdk-pixbuf cairo
-    mime-types pango gtk3
-    glib-networking gsettings-desktop-schemas
-    xclip notify-osd enchant
```

Comment:
`xclip` in `buildInputs` doesn't have any effect because it's a CLI tool, not a library. I've added `xclip` to the wrapper instead.

Reply:
`notify-osd` is removed since it's no longer needed. Details in  https://github.com/NixOS/nixpkgs/pull/125113#issue-658460773.